vuepress/docs/next/docs/consume/entandoapp-cr.md

```diff
@@ -27,7 +27,7 @@ spec:
 | :- | :- |
 | `spec.customServerImage`|  Used to deploy the Docker image containing your custom Entando App. Follow these instructions on how to [build your own image](../../tutorials/devops/build-core-image.md). This property and the `spec.standardServerImage` are mutually exclusive.|
 |`spec.dbms` | Allowed values are: MySQL, PostgreSQL (default), Oracle, or embedded. Oracle is only supported as an external database.|
-|`spec.ecrGitSshSecretName`| The configuration used by the Entando Component Repository to download bundles from authenticated Git repositories. It's a Secret containing a private key file named `rsa_id` that matches a public key configured in the authenticated Git repository.|
+|`spec.ecrGitSshSecretName`| The configuration used by the Local Hub to download bundles from authenticated Git repositories. It's a Secret containing a private key file named `rsa_id` that matches a public key configured in the authenticated Git repository.|
```

Comment:
Local Hub -> Entando Component Manager. In this case, it actually is the CM that does the download.